### ISSUE TITLE
Google docs forms multiple choice buttons don't work

### ISSUE THREAD

**Issue by jdm** [servo/servo#16694]
https://docs.google.com/forms/d/e/1FAIpQLSfjh3y2cmd1bSTpxRjXrdue_dIg0v2eIZD-AQtTCAw4kNo1ng/viewform?c=0&w=1

**Comment by Manishearth** [comment]
(They seem to be pure JS implementations)

**Comment by jdm** [comment]
Yeah, everything is a div with attributes like `jsaction` and `jscontroller` and `role`. I didn't see any smoking guns, and no JS errors were reported by Servo.

**Comment by Manishearth** [comment]
Did you try with the featuredetectordetector?

**Comment by jdm** [comment]
Nope!

**Comment by krk** [comment]
This can be reproduced in a14b952fa3eedec86e6308213c525538bcb20827

**Comment by mrobinson** [comment]
This still seems like an issue with a recent build.
